modules/react/form-field/stories/visual-testing/stories_Radio.tsx

```diff
@@ -126,31 +136,43 @@ export const RadioStates = () => (
         ]}
       >
         {props => (
-          <FormField
-            useFieldset={true}
-            hintText={typeof props.error !== 'undefined' ? hintText : undefined}
-            hintId={hintId}
-            {...props}
-          >
+          <FormField as="fieldset" {...props} cs={{width: props.grow ? '100%' : undefined}}>
+            <FormField.Label>{props.label}</FormField.Label>
             {testGroup}
+            {props.orientation === 'horizontal' ? (
+              <FormField.Container>
+                {testGroup}{' '}
+                {typeof props.error !== 'undefined' ? (
+                  <FormField.Hint>hintText</FormField.Hint>
+                ) : undefined}
+              </FormField.Container>
+            ) : undefined}
```

Comment:
```suggestion
            {props.orientation === 'horizontal' && (
              <FormField.Container>
                {testGroup}{' '}
                {props.error && (
                  <FormField.Hint>hintText</FormField.Hint>
                )}
              </FormField.Container>
            )}
```

React handles all falsy values and do not render anything for them, so `undefined` is not really needed here